### ISSUE TITLE
Another bug in kalman_filter.py about matrix dimensions

### ISSUE THREAD

**Issue by CuiiGen** [nasa/prog_algs#225]
https://github.com/nasa/prog_algs/blob/8689c09d125a761ac5e5f3394d290a661f13eb8d/src/prog_algs/state_estimators/kalman_filter.py#L67-L74

Thanks for you attention to the issue #221 that I submitted yesterday, and I am quite surprised by your efficiency in fixing this bug.
Here I come again because I find another problem about the dimension of matrix.

To account for the constant matrix $E$ in the states transaction equation $\dot{x}=Ax+Bu+E$, this equation can be transformed to 
```math
\dot{x}=Ax+\left[\begin{matrix}B&E\end{matrix}\right]\left[\begin{matrix}u\\1\end{matrix}\right]
```
So the constant `np.array([[1]])` should be appended to the tail of the variable `inputs`.
Apparently there is something wrong in the Line 73 above since the matrix E should be appended to B along with the axis of 1 rather than 0.
```python
B = np.append(B, deepcopy(model.E), 1) 
```
I do think ProgPy helps people who are studing or researching in the filed of Model Baed Pronostics a lot.
Hope that you guys have time, money, effort, and proper plans to test and validate your code.

**Comment by teubert** [comment]
Hello @CuiiGen 

Thanks for opening this issue. Most of my team is out for the Thanksgiving holiday, but we will take a look at this when we return next week.

Also, thank you for your kind words!
I'm glad that you find ProgPy useful.

**Comment by teubert** [comment]
Thank you @CuiiGen for identifying this issue. This was resolved in prog_algs v1.4.3, just released. 

Let me know if you run into any more issues or have any additional suggestions.
